astropy/cosmology/tests/test_cosmology.py

```diff
@@ -44,6 +44,45 @@ def test_flat_z1():
                     [2404.0, 2404.24, 2404.4] * u.Mpc, rtol=1e-3)
 
 
+@pytest.mark.skipif(not HAS_SCIPY, reason="requires scipy.")
+def test_varyde_lumdist_mathematica():
+    """Tests a few varying dark energy EOS models against a Mathematica computation."""
+
+    # w0wa models
+    z = np.array([0.2, 0.4, 0.9, 1.2])
+    cosmo = flrw.w0waCDM(H0=70, Om0=0.2, Ode0=0.8, w0=-1.1, wa=0.2, Tcmb0=0.0)
+    assert allclose(cosmo.w0, -1.1)
```

Comment:
Nope. I just didn't change the test. I actually think removing the checks is alright, as well, since each is checked in e.g. https://github.com/astropy/astropy/blob/2984fa2ea5f3604b71df8ca687c59d34913e62f1/astropy/cosmology/tests/test_flrw.py#L132

Reply:
All we really care about is comparing to the Mathematica calcs.

Reply:
I'm removing the parameter checks.
@mhvk , still alright to merge?

Reply:
Yes, definitely. Cleaner if this just checks that the calculations are OK.